sd-gui/src/app.rs

```diff
@@ -294,6 +304,7 @@ impl eframe::App for App {
 
                 ui.menu_button("Language", |ui| {
                     ui.radio_value(&mut self.language, UiLanguage::Chil, "Chil");
+                    ui.radio_value(&mut self.language, UiLanguage::LlvmIr, "LlvmIr");
```

Comment:
Probably applies in some other places but I believe the IR is usually referred to as "LLVM IR" in all caps (side note that the same applies for MLIR)